### DISCUSSION TITLE
[NEW HARDWARE] CC2674 (CC2674P/CC2674R) and CC1354P (CC1354P) Zigbee Coordinator and Zigbee Router adapters?

### DISCUSSION THREAD

**Discussion by Hedda** [tube0013/tube_gateways#161] [Ideas]
@tube0013 You considered making CC2674/CC2674P/CC2674R and/or CC1354/CC1354P Zigbee adapters for devs & early-adopters?

Check out this discussion -> https://github.com/Koenkk/Z-Stack-firmware/discussions/476

There is no community firmware from Koenkk yet but that is probably partially because no one made a CC2674 or CC1354 adapter yet.

**Comment by Hedda** [comment]
@tube0013 FYI, E72-2G4M20S1C (CC2674P10) radio module from Ebyte/Cdebyte is now available:

https://www.cdebyte.com/products/E72-2G4M20S1C

According to [LSagiroglu](https://github.com/LSagiroglu) this new CC2674P module is is pin to pin compatible with their CC2652P module:

https://www.cdebyte.com/products/E72-2G4M20S1E

PS: No news so far from RFstar if they are working on CC2674P or CC1354P radio modules yet.

**Comment by LordNex** [comment]
Take a look at this guy and thread. Would be interesting if we could somehow combine the 2

https://github.com/justLV/onju-voice/discussions/21

**Reply by tube0013** [reply]
https://github.com/tube0013/tube_gateways/assets/11084412/b468d959-8654-4336-8b1a-a4a564541eb1

**Reply by LordNex** [reply]
> 
> https://github.com/tube0013/tube_gateways/assets/11084412/b468d959-8654-4336-8b1a-a4a564541eb1

My kind of challenge. I'm just wondering if there's the ability to add a Zigbee repeater to it as well. So far I've gotten enough people together to put in an order for 10 prebuilt boards. But we're waiting for the BOM to come down a bit. Each one of us who puts an order in gets a different BOM for one. And none of us are sure about some of the options for build that it gives you. We're hoping to keep the price around $30-40 total per board. I've got a MuseLUX with RaspiAudio but it doesn't really mesh well on a wall or other places. Not to mention it's nice to be able to reuse something and reduce the obsolescence of everything so fast.

**Reply by tube0013** [reply]
No way to add a repeater with the hardware as designed and even this hardware is beyond the scope of my abilities. Plus collocating zigbee and WiFi is just a bad idea.

**Reply by LordNex** [reply]
Yeah I guess having both radios together wouldn't be very good. Was a thought. I fhinknikm going to end up hedging 5 prefab boards myself. He has a walk through out there. One think you loose is 1 of the 3 microphones in it but keeps all the other features. Add that with ChatGPT Running with a TPU and I think you could do some pretty star trek shit 🖖

**Reply by Hedda** [reply]
@LordNex off-topic but if you would base a such device on the new ESP32-C6 SoC instead then it could probably be made to also Zigbee Router device, but again off-topic so better if you discuss that in that other repository.
